html/elements/input/datetime-local.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-datetime-local": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/datetime-local",
+            "support": {
+              "chrome": {
+                "version_added": "20"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
```

Comment:
I would omit "This feature is not implemented yet." as it is redundant. Also the project page link doesn't seem very active, it's probably better to just link to the bug:

    "See <a href='https://bugzil.la/888320'>bug 888320</a>."